lua/gt.lua

```diff
@@ -16,8 +16,13 @@ return function (net_conf, numa_table)
 		n_lcores)
 	gatekeeper.gt_assign_lcores(gt_conf, gt_lcores)
 
+	local net_prefix = "10.0.2.1/24"
+	local net_prefix6 = "2003:db8::1/32"
+	gt_conf.max_num_ipv6_neighbors = 65535
```

Comment:
I didn't find such heuristics like we had for others, e.g., memory pool. The number here is just for example.